wgpu/src/util/mod.rs

```diff
@@ -19,9 +18,8 @@ use std::{
 pub use belt::StagingBelt;
 pub use device::{BufferInitDescriptor, DeviceExt};
 pub use encoder::RenderEncoder;
-pub use indirect::*;
 pub use init::*;
-pub use wgt::math::*;
+pub use wgt::{math::*, DispatchIndirectArgs, DrawIndexedIndirectArgs, DrawIndirectArgs};
```

Comment:
Technically a breaking change due to type name changes (`Arg` was added to each type name)

Reply:
Yeah, I should probably switch it back, was trying not to touch `core` though, but this obviously isn't going to land before arcinization :)

Reply:
Decided to keep the change, but mention it